integration-tests/firebase-admin/firebase.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Both `firebase.json` and `.firebaserc`are needed by the Emulator Suite.

Reply:
OK